jpegenc/subblocks/bytestuffer/bytestuffer.py

```diff
@@ -0,0 +1,179 @@
+"""
+This module is MyHDL implementation
+of Byte Stuffer used for JPEG Encoder
+"""
+
+from myhdl import always_seq, always_comb
+from myhdl import Signal, concat, intbv, block
+
+
+class BSInputDataStream(object):
```

Comment:
@cfelton . Ultimately these are going to be fifo interfaces, when we connect the individual modules.
i.e the input bus to `present module` is the output bus of the `previous modules's FIFO`.